instruction.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="instruction.css" crossorigin="anonymous">
+    <title>Instruction</title>
+</head>
+<center>
+<body bgcolor="lightgreen">
+    <h1>
+        <strong>Instruction</strong>
+    </h1>
+
+    <a href="index.html"><img src="images/back.png" class="leftarrow" style="width:50px;height:50px;"></a>
+
+    <div id="div1">
+    <table id="table1">

```

Comment:
You shouldnt need two tables for this. Try using the table header th element instead. https://www.w3schools.com/html/html_table_headers.asp